eng/common/scripts/git-branch-push.ps1

```diff
@@ -27,25 +27,30 @@ param(
     [Parameter(Mandatory = $false)]
     [string] $PushArgs = "",
 
+    [string] $RemoteName = "azure-sdk-fork",
+
+    [Parameter(Mandatory = $false)]
+    [boolean] $SkipCommit = $false,
+
     [Parameter(Mandatory = $false)]
-    [boolean] $SkipCommit = $false
+    [boolean] $AmendCommit = $false
 )
 
 # This is necessay because of the janky git command output writing to stderr.
 # Without explicitly setting the ErrorActionPreference to continue the script
 # would fail the first time git wrote command output.
 $ErrorActionPreference = "Continue"
 
-Write-Host "git remote add azure-sdk-fork $GitUrl"
-git remote add azure-sdk-fork $GitUrl
+Write-Host "git remote add $RemoteName $GitUrl"
+git remote add $RemoteName $GitUrl
```

Comment:
We should consider one of the options in https://stackoverflow.com/questions/57935486/add-git-remote-only-if-it-doesnt-exist instead of trying to parameterize and guess an unused remote name.

Reply:
I just did a `git remote` to check the existing remote.